src/pages/docs/tutorials/tutorial-hello-enact/basics/index.md

```diff
@@ -98,7 +98,7 @@ return (
 	</div>
 );
 ```
-You'll notice that by introducing `<div>` we no longer have valid JavaScript! In fact, it looks a lot like valid HTML. That's because React introduces [JSX](https://facebook.github.io/react/docs/jsx-in-depth.html), which is a JavaScript syntax extension. In order to make JSX runnable by the browser, it has to be converted to JavaScript. With Enact, this is handled during the build process using [webpack](http://webpack.github.io) and [babel](http://babeljs.io). More on this [later](#running-with-npm).
+You'll notice that by introducing `<div>` we no longer have valid JavaScript! In fact, it looks a lot like valid HTML. That's because React introduces [JSX](https://reactjs.org/docs/jsx-in-depth.html), which is a JavaScript syntax extension. In order to make JSX runnable by the browser, it has to be converted to JavaScript. With Enact, this is handled during the build process using [webpack](http://webpack.github.io/) and [babel](https://babeljs.io/). More on this [later](#running-the-app).
```

Comment:
```suggestion
You'll notice that by introducing `<div>` we no longer have valid JavaScript! In fact, it looks a lot like valid HTML. That's because React introduces [JSX](https://reactjs.org/docs/jsx-in-depth.html), which is a JavaScript syntax extension. In order to make JSX runnable by the browser, it has to be converted to JavaScript. With Enact, this is handled during the build process using [webpack](http://webpack.github.io) and [babel](https://babeljs.io). More on this [later](#running-the-app).
```

Reply:
It will be fixed.